fog/ledger/server/src/bin/ledger_router.rs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
```

Comment:
I originally left it as `ledger_router` because it was also the binary name but I see that we're renaming the `ledger_server` binary so I'll change this to match that, that's better anyway.